### PR TITLE
Swallow follower resolve errors, allow reading follower count unauthed

### PR DESCRIPTION
**Pull request by RangerMauve**
Hotfix for https://github.com/hyphacoop/distributed-press-organizing/issues/169

Just gonna swallow resolve errors for now. Longer term we should delete missing accounts and do better caching.

Also added https://github.com/hyphacoop/distributed-press-organizing/issues/137 while I'm at it. Just the count not the items for now

### CODE REVIEW

File: src/server/apsystem.ts [hunk 1]
```diff
@@ -488,16 +493,25 @@ export default class ActivityPubSystem {
...
+      // Filter out failed loads
+    )).filter((item) => item.length !== 0)
 
     return {
       '@context': 'https://www.w3.org/ns/activitystreams',
       type: 'OrderedCollection',
       id: followersURL,
       items,
-      totalItems: items.length
+      totalItems: followers.length
```
Comment (fauno) [inline src/server/apsystem.ts:514]: won't this produce innacurate counts?

File: src/server/api/followers.ts [hunk 2]
```diff
@@ -23,11 +23,14 @@ export const followerRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityP
   }, async (request, reply) => {
     const { actor } = request.params
     const allowed = await apsystem.hasPermissionActorRequest(actor, request)
+    const collection = await apsystem.followersCollection(actor)
+
+    // Allow seeing follower count but not followers
+    // TODO: Allow list to be public?
     if (!allowed) {
-      return await reply.code(403).send('Not Allowed')
+      delete collection.items
```
Comment (fauno) [inline src/server/api/followers.ts:31]: if i understand correctly, the followers are still webfingered but not shown publicly, right?

Comment (RangerMauve) [reply src/server/api/followers.ts:31]: In this case yes. I suppose I could just show the count though?

File: src/server/apsystem.ts (hunk 1 above)
Comment (RangerMauve) [reply src/server/apsystem.ts:514]: I think Mastodon does a similar thing where it shows the total count the user reports but doesn't ingest all the followers

Comment (fauno) [reply src/server/apsystem.ts:514]: As long as we document it's the currently active followers list I'm ok with this number

File: src/server/api/followers.ts (hunk 2 above)
Comment (fauno) [reply src/server/api/followers.ts:31]: I think it'll be faster and we can avoid launching lots of webfinger requests just to get an accurate count

**Review by fauno — APPROVED** [review]
Thanks!

**Pull request merged**